polkadot-wiki/sidebars.js

```diff
@@ -26,84 +26,108 @@ module.exports = {
       type: 'category',
       label: 'Learn',
       items: [
-        'learn/learn-launch',
         {
           type: 'category',
           label: 'Basics',
-          items: [
-            'learn/learn-architecture',
-            'learn/learn-accounts',
-            'learn/learn-account-generation',
-            'learn/learn-account-restore',
-            'learn/learn-polkadotjs',
-            'learn/learn-assets',
-            'learn/learn-nft',
-            'learn/learn-DOT',
-            'learn/learn-security',
-            'learn/learn-consensus',
-            'learn/learn-nominator',
-            'learn/learn-staking-faq',
-            'learn/learn-validator',
-            'learn/learn-collator',
-            'learn/learn-governance',
-            'learn/learn-identity',
-            'learn/learn-balance-transfers',
-            'learn/learn-transaction-fees',
-            'learn/learn-polkadot-host',
-            'learn/learn-treasury',
-            'learn/learn-registrar',
-            'learn/learn-runtime-upgrades',
-          ],
-        },
-        {
-          type: 'category',
-          label: 'Paraobjects',
           items: [
             {
               type: 'category',
-              label: 'Parachains',
+              label: 'Accounts',
               items: [
-                'learn/learn-parachains',
-                'learn/learn-common-goods',
-                'learn/learn-auction',
-                'learn/learn-crowdloans',
-                'learn/learn-teleport',
+                  'learn/learn-accounts',
+                  'learn/learn-account-generation',
+                  'learn/learn-account-restore',
+                  'learn/learn-balance-transfers',
+              ],
+            },
+            {
+              type: 'category',
+              label: 'Assets',
+              items: [
+                  'learn/learn-assets',
+                  'learn/learn-DOT',
+                  'learn/learn-nft',
+              ],
+            },
+            {
+              type: 'category',
+              label: 'Components',
+              items: [
+                  'learn/learn-architecture',
+                  'learn/learn-collator',
+                  'learn/learn-consensus',
+                  'learn/learn-governance',
+                  'learn/learn-identity',
+                  'learn/learn-nominator',
+                  'learn/learn-polkadot-host',
+                  'learn/learn-polkadotjs',
+                  'learn/learn-registrar',
+                  'learn/learn-security',
+                  'learn/learn-transaction-fees',
+                  'learn/learn-treasury',
+                  'learn/learn-runtime-upgrades',
+                  'learn/learn-validator',
+              ],
+            },
+            {
+              type: 'category',
+              label: 'Staking',
```

Comment:
Staking > Staking 

But I do not know what to name it. May be Staking Concepts > Staking? Also, this article is moved from advanced section to basics. We may need a more simpler doc for staking, but that is a discussion for sometime later. I think the placement of this doc in basics is fine.

Reply:
Was thinking the same, we have some docs in the basic section that are perhaps at the same level in some areas, but this doc still dives into details that aren't necessarily basics